Corzbank.Data/Entities/Transfer.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Corzbank.Data.Enums;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace Corzbank.Data.Entities
+{
+    public class Transfer: KeyEntity
+    {
+        [Required]
+        public TransferType TransferType { get; set; }
+       
+        public string ToPhoneNumber { get; set; }
```

Comment:
Rename to receiver phone number for consistence with receiverCardId

Reply:
resolved